llm/llama.go

```diff
@@ -84,6 +84,7 @@ func chooseRunners(workDir, runnerType string) []ModelRunner {
 	case "windows":
 		// TODO: select windows GPU runner here when available
 		runners = []ModelRunner{
```

Comment:
In this file you will also want to enable dynamic NumGPU allocation in the NumGPU function by doing the VRAM check in the case of windows also. Something like this:
```
index 7172f91..85540e8 100644
--- a/llm/llama.go
+++ b/llm/llama.go
@@ -269,7 +269,7 @@ func NumGPU(numLayer, fileSizeBytes int64, opts api.Options) int {
        if opts.NumGPU != -1 {
                return opts.NumGPU
        }
-       if runtime.GOOS == "linux" {
+       if runtime.GOOS == "linux" || runtime.GOOS == "windows" {
                freeBytes, err := CheckVRAM()
                if err != nil {
                        if !errors.Is(err, errNvidiaSMI) {
```

Reply:
Fixed, thx